### PR TITLE
Get started codelab: copyedits and markdown cleanup

### PR DESCRIPTION
**Pull request by chalin**
Mainly markdown cleanup. There are a few exceptions that I'll point out using inlined comments. It is probably best to [review this PR by ignoring whitespace](https://github.com/flutter/website/pull/1123/files?w=1).

Staged, see:

- https://flutter-io-staging-1.firebaseapp.com/get-started/codelab
- https://flutter-io-staging-1.firebaseapp.com/get-started/learn-more

cc @Sfshaza @kwalrath

### CODE REVIEW

**Comment by chalin** [comment]
I found mini "templates" for alert formatting. I've moved them into `_config.yml` so that they are slightly more convenient to use. They offer a form of lightweight Jekyll filter.

One that note, the _tip_ used a checkbox icon rather than a lightbulb. Let me know which you prefer.

File: _config.yml [hunk 1]
```diff
@@ -18,3 +18,30 @@ exclude: [example, packages, tool]
 
 plugins:
   - jekyll-redirect-from
+
+## Site-wide shorthands
+
+alert:
+  tip: >- # Use fa-lightbulb-o for tip?
+    <aside class="alert alert-success" role="alert" markdown="1">
+    <i class="fa fa-check-square-o"></i> **Tip:**
```
Comment (chalin) [inline _config.yml]: For tips, do you prefer an `fa-check-square-o` or an `fa-lightbulb-o` icon?

File: get-started/codelab/index.md [hunk 2]
```diff
@@ -207,92 +206,83 @@ You can find the
...
-flutter packages get
-Running "flutter packages get" in startup_namer...
-Process finished with exit code 0
-{% endprettify %}
-</li>
+    ```terminal
+    > flutter packages get
+    Running "flutter packages get" in startup_namer...
+    Process finished with exit code 0
+    ```
```
Comment (chalin) [inline get-started/codelab/index.md]: Is the use of this new terminal styling ok (set green on black)?

File: get-started/codelab/index.md [hunk 3]
```diff
@@ -973,6 +933,7 @@ In this codelab, you've:
   and the new route.
 * Learned about changing the look of your app's UI using Themes.
 
-<p class="h2-like">Next step</p>
+## Next step
+{:.no_toc}
 
-[Next step: Learn More](/get-started/learn-more/)
+[Learn More](/get-started/learn-more/)
```
Comment (chalin) [inline get-started/codelab/index.md]: Note: I dropped the "Next step:" prefix here.

File: _config.yml (hunk 1 above)
Comment (Sfshaza) [reply _config.yml]: I was the one who used lightbulb originally, but am flexible.

File: get-started/codelab/index.md [hunk 4]
```diff
@@ -85,16 +85,15 @@ You’ll be modifying this starter app to create the finished app.
 In this codelab, you'll mostly be editing **lib/main.dart**,
 where the Dart code lives.
 
-<aside class="alert alert-success" markdown="1">
-  <i class="fa fa-lightbulb-o"></i> **Tip:**
+{{site.alert.tip}}
```
Comment (Sfshaza) [inline get-started/codelab/index.md]: Nice.

File: get-started/codelab/index.md [hunk 5]
```diff
@@ -138,18 +137,18 @@ where the Dart code lives.
   of Material widgets.
 - The main method specifies fat arrow (`=>`) notation,
   which is short hand used for one-line functions or methods.
-- The app extends StatelessWidget which makes the app itself a
+- The app extends `StatelessWidget` which makes the app itself a
```
Comment (Sfshaza) [inline get-started/codelab/index.md]: We usually don't codefont class names, but it IS Google style guidelines, so it's fine.

File: get-started/codelab/index.md [hunk 6]
```diff
@@ -207,92 +206,83 @@ You can find the
...
     In **pubspec.yaml**, add **english_words** (3.1.0 or higher)
     to the dependencies list.
     The new line is highlighted below:
 
-<!-- skip -->
-{% prettify dart %}
-dependencies:
-  flutter:
-    sdk: flutter
+    {% prettify yaml %}
```
Comment (Sfshaza) [inline get-started/codelab/index.md]: Let's talk about prettify offline. I was using prettify, as I always do, when I realized that few of the engineers have been using it (most docs use tick-tick-tick), and the TVC tech writer made a passionate argument for not using prettify, unless she needs code highlighting. I finally gave in.

File: get-started/codelab/index.md (hunk 2 above)
Comment (Sfshaza) [reply get-started/codelab/index.md]: If it's consistent with dartlang, that's fine. (I haven't seen it yet, though.)

File: get-started/codelab/index.md [hunk 7]
```diff
@@ -333,94 +322,87 @@ In this step, you’ll add a stateful widget, RandomWords, which creates
...
+          ),
+        );
+      }
+    }
+    {% endprettify %}
 
 Restart the app. If you try to hot reload, you might see a warning:
 
-{% prettify sh %}
+```nocode
```
Comment (Sfshaza) [inline get-started/codelab/index.md]: Noted (nocode). New to me.

File: get-started/codelab/index.md [hunk 8]
```diff
@@ -447,151 +430,143 @@ displayed in a ListView widget, grows infinitely. ListView's
...
+    {% endprettify %}
 
 Restart the app. You should see a list of word pairings. Scroll down
 as far as you want and you will continue to see new word pairings.
 
 <center><img src="images/step4-screenshot.png" alt="screenshot at completion of fourth step"></center>
 
-<p class="h2-like">Problems?</p>
+## Problems?
+{:.no_toc}
```
Comment (Sfshaza) [inline get-started/codelab/index.md]: Also nice. (Didn't remember this one.)

File: get-started/learn-more.md [hunk 9]
```diff
@@ -3,28 +3,26 @@ layout: page
 title: "Get Started: Learn More"
 permalink: /get-started/learn-more/
 ---
-* Learn more about the Flutter framework
 
-{% comment %}
+Learn more about the Flutter framework:
+
+{%- comment %}
 Once the "Flutter for React" doc (etc) are done, add here.
```
Comment (Sfshaza) [inline get-started/learn-more.md]: Oooh, this is ready! Could you add links for "Flutter for React Native", "Flutter for iOS", "Flutter for Xamarin.Forms" here?

File: _config.yml (hunk 1 above)
Comment (chalin) [reply _config.yml]: Any preference? (I have no preference atm.)

File: get-started/codelab/index.md (hunk 2 above)
Comment (chalin) [reply get-started/codelab/index.md]: Yes it is consistent with dartlang, e.g., see https://www.dartlang.org/tools/sdk#install

File: get-started/codelab/index.md (hunk 6 above)
Comment (chalin) [reply get-started/codelab/index.md]: > using prettify, unless she needs code highlighting

I agree with that statement :). Markdown fenced code blocks are processed by [Rouge](https://github.com/jneen/rouge) at site compile time, which I think is nice.

I'm unsure what the objections were to prettify, but I've improved the plugin a bit (and can make further changes as necessary).

File: _config.yml (hunk 1 above)
Comment (chalin) [reply _config.yml]: I've switch it (back) to fa-lightbulb-o.

File: get-started/learn-more.md (hunk 9 above)
Comment (chalin) [reply get-started/learn-more.md]: Adding them. What about "Flutter for Android Developers"?

Comment (chalin) [reply get-started/learn-more.md]: Done, including "Flutter for Android", sorted alphabetically.

**Comment by chalin** [comment]
Updates done. Staging server refreshed. PTAL

File: get-started/codelab/index.md (hunk 2 above)
Comment (Sfshaza) [reply get-started/codelab/index.md]: Then leave it.  :)

**Comment by Sfshaza** [comment]
For tips, I really do prefer the lightbulb.

**Comment by chalin** [comment]
> For tips, I really do prefer the lightbulb.

Done in 33907155b0cc418aaae625b333cb4279f39085ad

**Comment by Sfshaza** [comment]
LGTM

**Pull request merged**